src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenReadonlyStructTests.cs

```diff
@@ -2528,5 +2528,39 @@ static void Main()
 ";
             CompileAndVerify(csharp, options: TestOptions.UnsafeReleaseExe, verify: Verification.Fails, expectedOutput: "42");
         }
+
+        [Fact]
+        public void ReadOnlyGetter_LangVersion()
+        {
+            var csharp = @"
+struct S
+{
+    public int P { get; }
+
+    static readonly S Field = default;
+    static void M()
+    {
+        _ = Field.P;
+    }
+}
+";
+
+            CompileAndVerify(csharp, parseOptions: TestOptions.Regular7_3, symbolValidator: validate7_3, verify: Verification.Passes);
+            CompileAndVerify(csharp, symbolValidator: validate, verify: Verification.Fails);
+
+            void validate7_3(ModuleSymbol module)
```

Comment:
>void [](start = 12, length = 4)

static? (same for `validate`)

Reply:
Rather than local functions and validation did you consider using a theory and different data sets? 

---
In reply to: [277839189](https://github.com/dotnet/roslyn/pull/35213#discussion_r277839189) [](ancestors = 277839189)

Reply:
Done